src/matching.rs

```diff
@@ -51,19 +51,17 @@ pub fn compute_matches_single_threaded<'a>(choices: &[&'a str], query: &str, fil
 
 pub fn compute_matches_multi_threaded<'a>(choices: &[&'a str], query: &str, filter_only: bool) -> Vec<&'a str> {
     use std::sync::mpsc::channel;
-    let mut join_guards = Vec::new();
     let (tx, rx) = channel();
     let workers = num_cpus::get();
     for current_worker in 0..workers {
         let tx = tx.clone();
-        let join_guard = crossbeam::scope(|_| {
+        crossbeam::scope(|_| {
```

Comment:
The usage of crossbeam here was actually not correct. When I fixed it, the `join_guards` vector became unnecessary.